src/authentication/authentication.controller.ts

```diff
@@ -44,8 +50,20 @@ export class AuthenticationController {
     response.setHeader('Set-Cookie', cookie);
   }
 
+  @UseGuards(JwtAuthenticationGuard)
+  @Patch('update-phone-number')
+  @TransformPlainToInstance(AuthenticationResponseDto)
+  async updatePhoneNumber(
```

Comment:
`updatePhoneNumber` has nothing to do with authentication. It should be in the `UsersController` instead